core/src/main/java/org/apache/iceberg/TableMetadataParser.java

```diff
@@ -83,30 +83,30 @@ private TableMetadataParser() {
   }
 
   // visible for testing
-  static final String FORMAT_VERSION = "format-version";
-  static final String TABLE_UUID = "table-uuid";
-  static final String LOCATION = "location";
-  static final String LAST_SEQUENCE_NUMBER = "last-sequence-number";
-  static final String LAST_UPDATED_MILLIS = "last-updated-ms";
-  static final String LAST_COLUMN_ID = "last-column-id";
-  static final String SCHEMA = "schema";
-  static final String SCHEMAS = "schemas";
-  static final String CURRENT_SCHEMA_ID = "current-schema-id";
-  static final String PARTITION_SPEC = "partition-spec";
-  static final String PARTITION_SPECS = "partition-specs";
-  static final String DEFAULT_SPEC_ID = "default-spec-id";
-  static final String LAST_PARTITION_ID = "last-partition-id";
-  static final String DEFAULT_SORT_ORDER_ID = "default-sort-order-id";
-  static final String SORT_ORDERS = "sort-orders";
-  static final String PROPERTIES = "properties";
-  static final String CURRENT_SNAPSHOT_ID = "current-snapshot-id";
-  static final String REFS = "refs";
-  static final String SNAPSHOTS = "snapshots";
-  static final String SNAPSHOT_ID = "snapshot-id";
-  static final String TIMESTAMP_MS = "timestamp-ms";
-  static final String SNAPSHOT_LOG = "snapshot-log";
-  static final String METADATA_FILE = "metadata-file";
-  static final String METADATA_LOG = "metadata-log";
+  public static final String FORMAT_VERSION = "format-version";
+  public static final String TABLE_UUID = "table-uuid";
+  public static final String LOCATION = "location";
+  public static final String LAST_SEQUENCE_NUMBER = "last-sequence-number";
+  public static final String LAST_UPDATED_MILLIS = "last-updated-ms";
+  public static final String LAST_COLUMN_ID = "last-column-id";
+  public static final String SCHEMA = "schema";
+  public static final String SCHEMAS = "schemas";
+  public static final String CURRENT_SCHEMA_ID = "current-schema-id";
+  public static final String PARTITION_SPEC = "partition-spec";
+  public static final String PARTITION_SPECS = "partition-specs";
+  public static final String DEFAULT_SPEC_ID = "default-spec-id";
+  public static final String LAST_PARTITION_ID = "last-partition-id";
+  public static final String DEFAULT_SORT_ORDER_ID = "default-sort-order-id";
+  public static final String SORT_ORDERS = "sort-orders";
+  public static final String PROPERTIES = "properties";
+  public static final String CURRENT_SNAPSHOT_ID = "current-snapshot-id";
+  public static final String REFS = "refs";
+  public static final String SNAPSHOTS = "snapshots";
+  public static final String SNAPSHOT_ID = "snapshot-id";
+  public static final String TIMESTAMP_MS = "timestamp-ms";
+  public static final String SNAPSHOT_LOG = "snapshot-log";
+  public static final String METADATA_FILE = "metadata-file";
+  public static final String METADATA_LOG = "metadata-log";
```

Comment:
I don't have concern making these public personally as they are generally speaking usable by engines if needeed, but open to other opinions.

Reply:
I think we can revert this. No need to test `TableMetadataParser`, just `LoadTableResponse`.

Reply:
Also, I just checked and it doesn't look like `SNAPSHOT_ID` is used in the tests. The string is embedded instead, so it should be even easier to make this change.